src/utils/source-maps.js

```diff
@@ -46,3 +49,9 @@ export async function getMappedLocation(
 
   return sourceMaps.getOriginalLocation(location, source);
 }
+
+export function getSelectedLocation(mappedLocation: MappedLocation, selectedSource: Source) {
+  return selectedSource && isGenerated(selectedSource)
```

Comment:
You don't need the `selectedSource` check since `selectedSource: Source` doesn't have a `?`, right?